app/helpers/mb_form_builder.rb

```diff
@@ -0,0 +1,71 @@
+class MbFormBuilder < ActionView::Helpers::FormBuilder
+  def mb_input_field(method, label_text, type: "text", notes: [], options: {}, classes: [], prefix: nil, autofocus: nil)
+    classes = classes.append(%w[text-input])
+    <<-HTML.html_safe
+      <fieldset class="form-group#{error_state(object, method)}">
+        #{label_and_field(method, label_text, text_field(method, { autofocus: autofocus, type: type, class: classes.join(' '), autocomplete: 'off', autocorrect: 'off', autocapitalize: 'off', spellcheck: 'false' }.merge(options)), notes: notes, prefix: prefix)}
+        #{errors_for(object, method)}
+      </fieldset>
+    HTML
+  end
+
+  def mb_date_select(method, label_text, notes: [], options: {}, autofocus: nil)
+    <<-HTML.html_safe
+      <fieldset class="form-group#{error_state(object, method)}">
+        #{label(method, label_contents(label_text, notes))}
+        <div class="input-group--inline">
+          <div class="select">
+            #{date_select(method, { autofocus: autofocus, date_separator: '</div><div class="select">' }.merge(options), class: 'select__element')}
+          </div>
+        </div>
+        #{errors_for(object, method)}
+      </fieldset>
```

Comment:
Yeah it is a lot! The good news is that I found these helper methods to be very user-friendly when creating forms for GCF.